express/code/blocks/ax-columns/ax-columns.css

```diff
@@ -701,6 +705,7 @@ main .ax-columns.highlight.dark .columns-video:last-of-type {
 .ax-columns.fullsize .hero-animation-overlay {
   padding-top: 4px;
   padding-bottom: 4px;
+  height: var(--hero-animation-overlay-video-height);
```

Comment:
I feel like using min-height should be safer, just in case weird content gets used without our notice